### DISCUSSION TITLE
Error while creating AdminClient for Cluster

### DISCUSSION THREAD

**Discussion by ZKANGGIT** [kafbat/kafka-ui#701] [Q&A]
![image](https://github.com/user-attachments/assets/44265c05-2e56-4ce4-a7ae-3de0f9084f84)
I used docker to deploy the kafka service on the 162 server and the 179 server respectively, kafka-ui is deployed on 162, but kafka-ui can't connect to 179, the corresponding requesthttp://*.*.*.162:8082/api/clusters returns an error message: "lastError": {
            "message": "Error while creating AdminClient for Cluster *.*.*.179",
            "stackTrace": "java.lang.IllegalStateException: Error while creating AdminClient for Cluster *.*.*.179ntat com.provectus.kafka.ui.service.AdminClientServiceImpl. lambda$createAdminClient$5(AdminClientServiceImpl.java:56)ntat reactor.core.publisher.Mono.lambda$onErrorMap$28(Mono.java:3783)ntat reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:94)ntat reactor.core.publisher.MonoPeekTerminal$MonoTerminalPeekSubscriber.onError(MonoPeekTerminal.java:258)ntat reactor.core.publisher.FluxMap$MapConditionalSubscriber.onError(FluxMap.java:265)ntat reactor.core.publisher.Operators$MonoSubscriber.onError(Operators.java:1886)ntat reactor.core.publisher.MonoCacheTime$CoordinatorSubscriber.signalCached( MonoCacheTime.java:3

**Comment by Haarolean** [comment]
Hi,
1) the version of the app you're running is unsupported
2) can't help further without the full stacktrace
